src/integrationTest/java/org/opensearch/test/framework/TestSecurityConfig.java

```diff
@@ -142,6 +144,17 @@ public TestSecurityConfig user(User user) {
         return this;
     }
 
+    public TestSecurityConfig withRestAdminUser(final String name, final String... permissions) {
```

Comment:
This is because we will have tests (I will add them as a second part) which test REST admin with a full list of permissions and REST Admin with only one permission.